.github/actions/build-to/action.yml

```
@@ -0,0 +1,33 @@
+name: 'Build workspaces'
+description: 'Runs the build command for specific workspaces.'
+inputs:
+  workspaces:
```

based on my previous comment, this should be rather singular
```suggestion
  workspace:
```

---

Because of the previous discussion (about this input being an array), I think we should keep it plural, ie `workspaces`.